config/routes.rb

```diff
@@ -27,6 +28,8 @@
   resources :doctors do
     get :dashboard, to: 'doctors#dashboard', as: 'dashboard'
     get :profile
+
+    resources :consultation_requests, controller: 'doctors/consultation_request'
```

Comment:
Ideally you could just remove the `controller: 'foo'` portion of this declaration - however since this is just a prototype, we can leave it as is